### PR TITLE
WIP: add K8s repairnator worker dockerimage and yaml files for worker and ActiveMQ

### PR DESCRIPTION
**Pull request by Tailp**
This PR contains the docker image for the K8s repairnator worker and also yaml files for deploying the worker and ActiveMQ on K8s :).

### CODE REVIEW

File: repairnator/kubernetes-support/repairnator-deployment-yamlfiles/repairnator-worker.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: repairnator-worker
```
Comment (monperrus) [inline repairnator/kubernetes-support/repairnator-deployment-yamlfiles/repairnator-worker.yaml]: rename repairnator-worker to repairnator-pipeline everywhere

**Comment by monperrus** [comment]
We don't maintain two pipeline Dockers, so remove `k8s-pipeline-dockerimage/` and reuse instead `pipeline-dockerimage/`

**Comment by monperrus** [comment]
In this pull request, we want three changes:

* the publisher
* the repairnator-pipeline.yml
* the markdown documentation on how to use both

**Comment by Tailp** [comment]
> 
> 
> We don't maintain two pipeline Dockers, so remove `k8s-pipeline-dockerimage/` and reuse instead `pipeline-dockerimage/`

I will address this issue first :).

**Comment by Tailp** [comment]
@monperrus Hi :) , ready for a quick check of the structure. After this comes the implemetation.

**Comment by monperrus** [comment]
LGTM. Should be an interface instead.

**Comment by Tailp** [comment]
> 
> 
> LGTM. Should be an interface instead.

Hi :) , it will be implement an interface already provided by ActiveMQ itself called MessageListener . So if we want to create similar in the future we can implement that in the future.

**Comment by Tailp** [comment]
#904 need to be closed first before this can go on :) .

**Comment by Tailp** [comment]
BuildListener interface added in repairnator-core :) . PR (#901) scanner will also implement this later .

**Comment by Tailp** [comment]
Close to open a new PR :) .

**Comment by monperrus** [comment]
The branch name and pull request title should match the branch content.

Here it should be

* the publisher.py
* the repairnator-pipeline.yml
* the markdown documentation on how to use both

**Comment by Tailp** [comment]
Will open a new branch instead. This branch is a mess :) .

**Pull request closed without merging**